src/csharp/SDK/Native/NativeMethods.cs

```diff
@@ -106,6 +106,15 @@ public enum k4a_stream_result_t
             CalibrationDeviceType target_camera,
             out Vector3 target_point3d);
 
+        [DllImport("k4a", CallingConvention = k4aCallingConvention)]
+        [NativeReference]
+        public static extern k4a_result_t k4a_calibration_color_2d_to_depth_2d(
+            [In] ref Calibration calibration,
+            ref Vector2 source_point2d,
+            k4a_image_t detph,
```

Comment:
>detph [](start = 24, length = 5)

depth_image

Reply:
Fixed! Thanks,

Reply:
Hi Tsukasa, seems the typo is still there, please fix the "detph" to "depth"

---
In reply to: [388012196](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/pull/1097#discussion_r388012196) [](ancestors = 388012196)